x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/__mocks__/request_responses.ts

```diff
@@ -224,6 +224,24 @@ export const getFindResultWithMultiHits = ({
   };
 };
 
+export const getImportRulesRequest = (payload?: Buffer): ServerInjectOptions => ({
+  method: 'POST',
+  url: `${DETECTION_ENGINE_RULES_URL}/_import`,
+  headers: {
+    'Content-Type': 'multipart/form-data; boundary=frank_is_awesome',
+  },
+  payload,
+});
+
+export const getImportRulesRequestOverwriteTrue = (payload?: Buffer): ServerInjectOptions => ({
+  method: 'POST',
+  url: `${DETECTION_ENGINE_RULES_URL}/_import?overwrite=true`,
+  headers: {
+    'Content-Type': 'multipart/form-data; boundary=frank_is_awesome',
```

Comment:
Optional: Boundary can be anything you want it to be, doesn't have to be strictly `frank_is_awesome` though I wouldn't say that's a bad thing. :-)

Either way I would suggest if it makes sense to pull the boundary out so it doesn't have to be repeated in the tests as a constant somewhere such as in one of the test mock test files? Something like:

```ts
const TEST_BOUNDARY=frank_is_awesome
```

Reply:
I pulled it out into a utils file under `__mocks__` . I changed it to 'test_boundary', but added a note in the PR so everyone still knows that the statement `frank_is_awesome` is valid!